api/app/controllers/spree/api/base_controller.rb

```diff
@@ -55,29 +55,29 @@ def authenticate_user
         return if @current_api_user
 
         if requires_authentication? && api_key.blank? && order_token.blank?
-          must_specify_api_key and return
+          must_specify_api_key && return
```

Comment:
it doesnt work :)

Reply:
Not even one spec detects problem?

Reply:
@bbonislawski let's add regression specs + change rubocop config to omit the `&&` -> `and` rule

Reply:
@damianlegawiec I excluded all controllers from AndOr cop. I'll try to add some regression specs